openapi/stellar-disbursement-platform/bundled.yml

```diff
@@ -2080,6 +2080,14 @@ definitions:
         description: Enables multi-user approval workflow for disbursements
         type: boolean
         example: true
+      sms_registration_message_template:
+        description: The custom SMS message template set for the registration invite. The default message isn't returned.
+        type: string
+        example: Lorem ipsum dolor sit amet, consectetur adipiscing elit.
+      otp_message_template:
+        description: The custom OTP message template set to send OTP code to the receiver complete the registration flow. The default message isn't returned.
+        type: string
+        example: Lorem ipsum dolor sit amet, consectetur adipiscing elit. The default message isn't returned.
```

Comment:
@CaioTeixeira95 can you delete 'The default message isn't returned' from the example?

I'm happy to write real examples if helpful.